playbooks/de-populate-consul.yaml

```diff
@@ -254,6 +254,9 @@
         - {key: "configs/{{environment_name}}/de/redis-port", value: "{{ redis_port }}"}
         - {key: "configs/{{environment_name}}/de/redis-db-number", value: "{{ redis_db_number }}"}
         - {key: "configs/{{environment_name}}/de/redis-password", value: "{{ redis_password }}"}
+        - {key: "configs/{{environment_name}}/de/grouper-id", value: "{{ apps.grouper_user }}"}
```

Comment:
Oops, you're right.  I saw the config there, I just didn't think to use it directly.